tests/platform/cloud/OAuth2AcceptanceDataStore.js

```diff
@@ -68,6 +76,30 @@ fluid.defaults("gpii.oauth2.dataStore.acceptanceData", {
                 selectedPreferences: { "": true },
                 revoked: false
             }
+        ],
+        credentialClientTokensIdSeq: 4,
+        credentialClientTokens: [
+            {
+                id: 1,
+                clientId: 5,
+                accessToken: "first_discovery_access_token",
+                revoked: false,
+                allowAddPrefs: true
+            },
+            {
+                id: 2,
+                clientId: 99,
+                accessToken: "non_existent_client",
+                revoked: false,
+                allowAddPrefs: true
+            },
+            {
+                id: 3,
+                clientId: 2,
```

Comment:
Changing this to clientId 5 should enable us to verify that we are testing that the access token has allowAddPrefs rather than the client (see comment on authServer).